### DISCUSSION TITLE
Confirmation pop-up when opening external links

### DISCUSSION THREAD

**Discussion by SkypLabs** [tutao/tutanota#4478] [Ideas]
Hi,

I would like to suggest adding a confirmation pop-up or similar mechanism to confirm the click action when opening an external link. This would prevent opening external links by mistake (especially on touchscreens), which would result in improving the overall security / privacy of Tutanota's users at the same time.

This feature is already present and enabled by default in the Proton Mail client:

![IMG_20220808_180618](https://user-images.githubusercontent.com/9932586/183462977-87c56070-9887-4e5b-a81e-fdfd8adce61f.jpg)

Let me know what you think about it.

Thanks!

**Comment by serrq** [comment]
Yes, I also have Proton Mail and this feature improve security.
